internal/internal_task_handlers.go

```diff
@@ -153,6 +165,20 @@ type (
 	}
 )
 
+func (t *workflowTask) getAutoConfigHint() *s.AutoConfigHint {
+	if t.task != nil {
+		return t.task.AutoConfigHint
```

Comment:
do we need to check whether `t.task.AutoConfigHint` is nil?

Reply:
good catch. I changed the order so it makes more sense.